compiler/rustc_error_messages/src/lib.rs

```diff
@@ -307,19 +307,15 @@ impl From<Cow<'static, str>> for SubdiagnosticMessage {
 pub enum DiagnosticMessage {
     /// Non-translatable diagnostic message.
     Str(Cow<'static, str>),
-    /// Translatable message which has already been translated eagerly.
+    /// Translatable message which has already been translated already.
```

Comment:
```suggestion
    /// Translatable message which has already been translated.
```
(avoids duplication of "already")

Reply:
Oops, fixed!